package.json

```diff
@@ -10,6 +10,7 @@
     "type": "git",
     "url": "git://github.com/kangax/compat-table.git"
   },
+  "files": ["raw-data"],
```

Comment:
let's please not use the "files" key - this is a dangerous way to exclude things from npm. In addition, there's no need to exclude anything imo - https://unpkg.com/compat-table@0.0.1/ already has a bunch of other stuff.